.taskcluster.yml

```diff
@@ -27,12 +27,14 @@ tasks:
           zip -r fxa_client_android.zip fxa-client &&
           zip -r fxa_client_android_deps.zip fxa-client-deps
       artifacts:
-        'public/bin/fxa-client/fxa_client_android.zip':
+        'public/bin/mozilla/fxa-client/fxa_client_android_{{ event.version }}.zip':
           type: 'file'
           path: '/repo/fxa_client_android.zip'
-        'public/bin/fxa-client/fxa_client_android_deps.zip':
+        'public/bin/mozilla/fxa-client/fxa_client_android_deps_{{ event.version }}.zip':
```

Comment:
I only have these variables to play with: organization, module, version and ext. Meaning I can't map anything to /fxa-client/.

organization: mozilla
module: fxa_client_android or fxa_client_android_deps
version: e.g. 0.1.0
ext: zip

So, maybe use this instead (no fxa-client dir):
`'public/bin/mozilla/fxa_client_android_{{ event.version }}.zip':`

So, I can use this pattern `'/[organisation]/[module]_[version].[ext]'`

Reply:
Actually, nevermind. I lied :).

I can use artifact as a variable too. So, this should work as is!